posters-overrides.json

```diff
@@ -5364,9 +5366,7 @@
    "number": 1894
   },
   {
-   "number": 1895,
-   "videochat": "<a href=\"https://meet.jit.si/bids-derivatives\" target=\"_bids-derivatives\">jitsi:bids-derivatives</a>",
-   "pdf": "https://cdn-akamai.6connex.com/645/1827/poster_15922429379118925.pdf"
+   "number": 1895
```

Comment:
Accidental reversion.

Reply:
oops, what should I do now? Anything I can do?

Reply:
```suggestion
   "number": 1895,
   "videochat": "<a href=\"https://meet.jit.si/bids-derivatives\" target=\"_bids-derivatives\">jitsi:bids-derivatives</a>",
   "pdf": "https://cdn-akamai.6connex.com/645/1827/poster_15922429379118925.pdf"
```